examples/python/phi3-qa.py

```diff
@@ -79,6 +88,7 @@ def main(args):
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(argument_default=argparse.SUPPRESS, description="End-to-end AI Question/Answer example for gen-ai")
     parser.add_argument('-m', '--model', type=str, required=True, help='Onnx model folder path (must contain config.json and model.onnx)')
+    parser.add_argument('-e', '--execution_provider', type=str, required=True, choices=["cpu", "cuda", "dml"], help="Execution provider to run ONNX model with")
```

Comment:
Could we use the same option between phi3v and phi3-qa for the ep?

Reply:
Haha probably because the `-p` is occupied by `--top_p`. I do like the idea of unifying the parameters across different scripts. Just need to think of a better name.